app/components/TransactionModal.tsx

```diff
@@ -38,12 +37,8 @@ export const TransactionModal = ({
     <Dialog open={isModalOpen(ModalId.WAITING_TRANSACTION)} onOpenChange={close}>
       <DialogContent>
         <DialogHeader>
-          <VisuallyHidden asChild>
-            <>
-              <DialogTitle>Transaction Modal</DialogTitle>
-              <DialogDescription />
-            </>
-          </VisuallyHidden>
+          <DialogTitle></DialogTitle>
+          <DialogDescription></DialogDescription>
```

Comment:
why?

the idea of having `DialogTitle` is not just for not having the warning but also to provide label for aria description. In this case we lose that, right?

Reply:
Found a solution to this. But I think we can simplify on the UI lib. Thanks for pointing this out again